README.md

```diff
@@ -1,2 +1,56 @@
-# community
-Longhorn Community
+# Longhorn Community
+
+Focuses on supporting the Longhorn Cloud native distributed block storage built on and for Kubernetes
```

Comment:
Not really necessary?... It's pretty obvious.

Reply:
This is necessary.  Without it the file is not valid markdown

Reply:
I meant `Focuses on supporting the Longhorn Cloud native distributed block storage built on and for Kubernetes`